src/Widgets/DisplayWidget.vala

```diff
@@ -64,4 +64,10 @@ public class Power.Widgets.DisplayWidget : Gtk.Grid {
             return false;
         });
     }
+
+    public bool show_percentage (bool show) {
```

Comment:
Why does this need to return a boolean (always true) which is never used?  I suggest making `void`.